### DISCUSSION TITLE
Plugins for custom property conversion

### DISCUSSION THREAD

**Discussion by gunnarbittersmann** [cssbattle/forum#239] [Show and tell]
Now that `--` stopped working in current browsers you might want to convert `--` to `--Q` and vice versa. (Named after one of my favorite characters in Star Trek who appears and disappears at will.) If you’re a pro user, you might find these plug-ins useful.

`--` to `--Q`:
```
function run(code) {
	return code.replace(/--([:,)])/g, '--Q$1');
}
```

`--Q` to `--`:
```
function run(code) {
	return code.replace(/--Q([:,)])/g, '--$1');
}
```

**Comment by chinchang** [comment]
Thanks for sharing this @gunnarbittersmann 👍🏼  You are awesome! 🔥
